### DISCUSSION TITLE
Telegraf

### DISCUSSION THREAD

**Discussion by sn0opy** [selfhst/icons#4] [New Icon Request]
https://www.influxdata.com/time-series-platform/telegraf/

Not sure if there's a proper .svg logo. Was only able to find a png: https://github.com/influxdata/telegraf/blob/master/assets/TelegrafTiger.png

**Comment by shollyethan** [comment]
Telegraf has been added. I was only able to find a PNG version as well - I'll keep an eye out for an SVG file so I can create a light version as well.
